lib/vmpooler/statsd.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems' unless defined?(Gem)
```

Comment:
This can go away. `rubygems` is defined and included by default in Ruby 1.9.1 and up. Throw this line away.